nrf_rpc: Initialize transports independently of each other (#832)
nrfconnect/sdk-nrfxlib: nrf_rpc/CHANGELOG.rst

* Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.

Changes
=======

> * The initialization of transport is independent of each other.

Reviewer: ```suggestion
* Different transport mechanisms are initialized independent of each other.
```
Hoping not to change the meaning here.

Author: This looks okay to me. Thanks.

> * Different transport mechanisms are initialized independent of each other.

Reviewer 2: Could you make this entry a bit more precise: e.g.:
```suggestion
* Added a possibility for transport in each group to initialize independently of other groups. This enables the RPC to work for groups which are ready earlier rather than waiting for all groups to initialize their transports.
```
or something similar.

Author: Thanks for the suggestion.
I have changed the second part of the entry. Because the `nrf_rpc_init` function waits for all groups to be initialized. After that it is possible to use RPC.

* Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.

Changes
=======

> * Added a possibility for transport in each group to initialize independently of other groups.
> * This enables the RPC to work for those groups which are ready, even though initialization of others may have failed.

Reviewer: ```suggestion
* Enabled independent initialization of transport in each group.
  This makes RPC available for the groups that are ready, even though initialization of others may have failed.
```
Not separate bullets, because this is very much cause and effect.

Author: Right.
